assets/src/application/ui/forms/ee-form/adapters/Form.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Form as AntdForm } from 'antd';
+import { FormProps } from 'antd/es/form';
+
+const Form: React.FC<FormProps> = ({ children, onSubmit }) => {
+	return (
+		<AntdForm layout='vertical' onSubmit={onSubmit}>
```

Comment:
maybe use `vertical` as the default value for a `layout` prop, that way we have the option if need be to use the other formats

Reply:
Sounds good

Reply:
Done